### DISCUSSION TITLE
This site can't provide a secure connection

### DISCUSSION THREAD

**Discussion by milestones95** [supabase/supabase#2577] [Questions] [answered]
Hello, 

I sometimes receive the error lutcrhnocwappsgimbuo.supabase.co sent an invalid response. ERR_SSL_PROTOCOL_ERROR. Are there certain conditions where wifi cannot connect to supabase? I had to get a VPN to work around this.

**Comment by awalias** [comment, marked as answer]
hey! this is an ongoing issue - https://github.com/supabase/supabase/issues/2096

some xfinity routers are mistakenly blocking supabase.co , you can use ref.supabase.in as your SUPABASE_URL in the client as a temporary work around

if you are, or know an xfinity customer - please report this issue to them, it will help resolve the issue faster 👍

**Reply by awalias** [reply]
https://twitter.com/supabase/status/1420236362469363719?s=20
